graylog2-web-interface/src/components/content-packs/ContentPackEdit.test.jsx

```diff
@@ -46,19 +38,16 @@ describe('<ContentPackEdit />', () => {
   const selectedEntities = { dashboard: [{ id: '111-beef', type: 'dashbaord' }] };
 
   const parameter = { title: 'Port', name: 'PORT', type: 'integer', default_value: '23' };
```

Comment:
Maybe it is beyond the scope of this PR, but maybe we should introduce a `Parameter` value class as well?

Reply:
I did a value class for Entity in a following PR already. And will continue to transfer at need. (Which will come sooner or later!)